### PR TITLE
Sui

### PR DESCRIPTION
**Pull request by enriquesouza**
Today:
- Sui (RPC + Balance + Fee) DONE | Signing (TODO) - WIP 

**Tests Signing and Broadcasting:**

https://suiscan.xyz/mainnet/tx/8UvJbtBx6z7eG8fBWkn3qde4CqVmhvStaEJmA73J6KWx

https://suiscan.xyz/mainnet/tx/BWNdbfLNVGo7THjHH4DAjzj3tJ4HfTwiJh2unrABv7vn

### CODE REVIEW

File: frontend/src/services/Blockchain/polkadot/BlockchainServicePolkadot.ts [hunk 1]
```diff
@@ -300,8 +300,8 @@ export class BlockchainServicePolkadot
         signedTx.rawTransaction
       );
 
-      console.log('txBroadcastedHash:', txBroadcastedHash);
-      console.log('txHash:', signedTx.transactionHash);
+      // console.log('txBroadcastedHash:', txBroadcastedHash);
```
Comment (johnnyluo) [inline frontend/src/services/Blockchain/polkadot/BlockchainServicePolkadot.ts:303]: could you please remove it?

Comment (enriquesouza) [reply frontend/src/services/Blockchain/polkadot/BlockchainServicePolkadot.ts]: Sure, removed.

File: frontend/src/vault/send/form/hooks/useIsSendFormDisabled.tsx [hunk 2]
```diff
@@ -45,9 +48,9 @@ export const useIsSendFormDisabled = () => {
       return t('send_invalid_receiver_address');
     }
 
-    if (receiver === sender) {
-      return t('same_sender_receiver_error');
-    }
+    // if (receiver === sender) {
```
Comment (johnnyluo) [inline frontend/src/vault/send/form/hooks/useIsSendFormDisabled.tsx:51]: can you please remove this?

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**